client/my-sites/exporter/guided-transfer-card/index.jsx

```diff
@@ -86,7 +90,7 @@ class GuidedTransferCard extends Component {
 							'site{{/strong}} to a self-hosted WordPress.org installation with ' +
 							'one of our hosting partners.', { components: { strong: <strong /> } }
 						) }
-						<br/>
+						<br />
 						<a href="https://en.support.wordpress.com/guided-transfer/" >
```

Comment:
I know nothing changed here but this should use the support lib at [lib/url/support.js](https://github.com/Automattic/wp-calypso/blob/master/client/lib/url/support.js)